common-npm-packages/packaging-common/universal/ArtifactToolUtilities.ts

```diff
@@ -11,7 +11,7 @@ import * as toollib from "azure-pipelines-tool-lib/tool";
 export function getArtifactToolLocation(dirName: string): string {
     let toolPath: string = path.join(dirName, "ArtifactTool.exe");
     if (tl.osType() !== "Windows_NT"){
-        toolPath = path.join(dirName, "Artifacttool");
+        toolPath = path.join(dirName, "ArtifactTool");
```

Comment:
You mentioned offline that a previous version of the task worked, I believe with the same version of ArtifactTool that was being used in the failing instance, with `artifacttool` all lowercase. If the file is all lowercase in the zip file, it needs to be all lowercase here too.

Reply:
That's a bit of a conundrum, though. Zheng was moving to `ArtifactTool` because other uses were complaining that all-lowercase was also not working.

Reply:
https://github.com/microsoft/azure-pipelines-tasks/pull/18143

Reply:
moving to all lowercase for that path segment